command/commands/user.py

```diff
@@ -75,25 +83,23 @@ def handle(self, command, user_id):
             # Going for the nuclear option
             args = self.parser.parse_args(command_arg)
         except SystemExit:
-            return self.help
+            return self.bot.send_to_channel(self.help, channel)
 
         if args.which is None:
-            return self.help
+            return self.bot.send_to_channel(self.help, channel)
 
         elif args.which == "view":
             # stub
             if args.slack_id is not None:
-                return args.slack_id
+                self.view_helper(user_id, args.slack_id, channel)
             else:
-                return user_id
+                self.view_helper(user_id, None, channel)
```

Comment:
These two calls to `view_helper` can be squashed together (the it statement doesn’t really do anything).